pkg/network/tracer/event.go

```diff
@@ -376,6 +390,30 @@ func newIPRouteDest(source, dest util.Address, netns uint32) *ipRouteDest {
 	return d
 }
 
+func getPortType(p uint16) network.EphemeralPortType {
+	initfunc := func() {
+		procfsPath := "/proc"
+		if config.Datadog.IsSet("procfs_path") {
+			procfsPath = config.Datadog.GetString("procfs_path")
+		}
+
+		ephemeralIntPair = sysctl.NewIntPair(procfsPath, "net/ipv4/ip_local_port_range", time.Hour)
+	}
+	initEphemeralIntPair.Do(initfunc)
```

Comment:
nit: you can define the function inline without assigning it to a variable:
`initEphemeralIntPair.Do(func() {
...
})`